File: aula#7/pizza/PizzaBrasileira.py
from PizzaFactory import PizzaFactory
from PizzaMandioca import PizzaMandioca
from PizzaCamarao import PizzaCamarao

# ConcretoFactory A
class PizzaBrasileira(PizzaFactory):

    def criar_pizza_veg(self):
        return PizzaMandioca()
    
    def criar_pizza(self):
        return PizzaCamarao()

File: aula#7/AbstractFactory_java/Pizza/PizzaBrasileira.java
package Pizza;

/*
 * PizzaBrasileira e PizzaItaliana: Fábricas concretas que implementam
 * a criação de pizzas específicas.
 */
//ConcreteFactory A
class PizzaBrasileira extends PizzaFactory {

 @Override
 public PizzaVeg criarPizzaVeg() {
     return new PizzaMandioca();
 }

 @Override
 public Pizza criarPizza() {
     return new PizzaCamarao();
 }
}
